complex/ft_test.c

```diff
@@ -479,7 +513,7 @@ static int ft_run_bandwidth(void)
 			return ret;
 		}
 
-		show_perf("bw", ft_tx_ctrl.msg_size, recv_cnt, &start, &end, 1);
+		show_perf("bw", ft_ctrl.size_array[i], recv_cnt, &start, &end, 1);
```

Comment:
When sending the message to signal completion, the send_msg function being used sends a message of size ft_tx_ctrl.msg_size which can grow too large so it is reset to a smaller size after RMAs are done to send the message. But the performance should show the RMA size so I changed it to the looped value (which is set at the beginning of the testing loop).